CompilerFolderHandling/New-BcCompilerFolder.ps1

```diff
@@ -195,17 +195,23 @@ try {
         Remove-Item -Path $dotnetFolder -Recurse -Force
     }
 
-    $containerCompilerPath = Join-Path $compilerFolder 'compiler'
-    if ($vsixFile) {
-        # If a vsix file was specified unpack directly to compilerfolder
+    $containerCompilerPath = Join-Path $compilerFolder 'compiler';
+
+	if ($vsixFile) {
+        # If a vsix "file" was specified it is an already extracted path
         Write-Host "Using $vsixFile"
-        $tempZip = Join-Path ([System.IO.Path]::GetTempPath()) "alc.$containerName.zip"
-        Download-File -sourceUrl $vsixFile -destinationFile $tempZip
-        Expand-7zipArchive -Path $tempZip -DestinationPath $containerCompilerPath
+
+		DeletePathOrSymbolicLink -path $containerCompilerPath;
+		New-Item -Path $containerCompilerPath -ItemType SymbolicLink -Value $vsixFile | Out-Null;
+
+		if ($cacheFolder) {
+			DeletePathOrSymbolicLink -path $compilerPath;

```

Comment:
Will the GitHub cache cache files behind the symbolic link?
cacheFolder here is the folder, which is cached to the GitHub cache if the workflow is successful - this needs to include the compiler as well?
I wouldn't expect it to - so maybe if we are extracting to a cacheFolder - we need to copy?